bbot/cli.py

```diff
@@ -34,13 +34,8 @@ async def _main():
     import traceback
     from contextlib import suppress
 
-    # fix tee buffering (only if on real TTY)
-    if hasattr(sys.stdout, "reconfigure"):
-        try:
-            if sys.stdout.isatty():
-                sys.stdout.reconfigure(line_buffering=True)
-        except Exception:
-            pass
+    # fix tee buffering
+    sys.stdout.reconfigure(line_buffering=True)
```

Comment:
this was necessary because certain things like pytest's mock stdout don't have a .reconfigure() method